src/middleware/validation.middleware.js

```diff
@@ -25,6 +25,7 @@ export default () => {
         if (schema[key]) {
           toValidate[key] = req[key];
         }
+        // console.log(req[key], '<><><><<<<>>>>>');
```

Comment:
You may want to remove all the lines you commented out, if you are not using them. Thank you